css/pages/home.css

```diff
@@ -1,17 +1,18 @@
 /* base */
-@import '../base/base.css';
-@import '../base/layout.css';
+@import "../base/base.css";
+@import "../base/layout.css";
 
 /* 3rd-party */
 
 /* components */
-@import '../components/button.css';
-@import '../components/form.css';
+@import "../components/button.css";
+@import "../components/form.css";
 
 /* CUSTOM STYLE GOES BELOW THIS COMMENT LINE */
 
 /* HEADER */
-/* enf of HEADER */
+@import "../components/header.css";
+/* end of HEADER */
```

Comment:
dvigubos kabutes. turi buti viengubos. Reikia susitvarkyti formateri (prettier)

Reply:
ok, pataisiau sita ir dar pora vietu suformatavau is naujo. Zodziu taip iseina reikia pereiti per visus css puslapius kiekviena karta ir suformatuot kiekviena atskirai del sventos ramybes, nes kazkaip vis islenda sitos bedeles.